cpp/pixels-index/pixels-index-rockset/io_pixelsdb_pixels_index_rocksdb_RocksetIndex.h

```diff
@@ -50,10 +50,10 @@ JNIEXPORT void JNICALL Java_io_pixelsdb_pixels_index_rocksdb_RocksetIndex_DBdele
 /*
  * Class:     io_pixelsdb_pixels_index_rocksdb_RocksetIndex
  * Method:    CloseDB0
- * Signature: (JJ)V
+ * Signature: (J)V
  */
 JNIEXPORT void JNICALL Java_io_pixelsdb_pixels_index_rocksdb_RocksetIndex_CloseDB0
-  (JNIEnv *, jobject, jlong, jlong, jlong);
```

Comment:
I notice there is a `io_pixelsdb_pixels_index_rocksdb_RocksetTest.h` in the same path. Please remove it if it is not useful.